src/locales/en.json

```diff
@@ -23,6 +23,7 @@
     "fontFamily": "Font family",
     "onlySelected": "Only selected",
     "withBackground": "With Background",
+    "addWatermark": "Add watermark (help spread the word!)",
```

Comment:
Not sure about: `(help spread the word!)`

Reply:
FWIW, I think it's as good or better than anything I've come up with so we should just go with it and let someone submit a PR if they have a better idea :)

Reply:
```suggestion
    "addWatermark": "Add watermark",
```